test/e2e/specs/site-editor/zoom-out.spec.js

```diff
@@ -3,65 +3,54 @@
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
 
+	test.beforeEach( async ( { admin, editor } ) => {
 		await admin.visitSiteEditor();
 		await editor.canvas.locator( 'body' ).click();
 	} );
 
-	test.afterEach( async ( { admin, page } ) => {
-		await admin.visitAdminPage( 'admin.php', 'page=gutenberg-experiments' );
-		const zoomedOutCheckbox = page.getByLabel(
-			'Enable zoomed out view when selecting a pattern category in the main inserter.'
-		);
-		await zoomedOutCheckbox.setChecked( false );
-		await expect( zoomedOutCheckbox ).not.toBeChecked();
-		await page.getByRole( 'button', { name: 'Save Changes' } ).click();
-	} );
-
-	test.afterAll( async ( { requestUtils } ) => {
-		await requestUtils.activateTheme( 'twentytwentyone' );
-	} );
-
-	test( 'Clicking on inserter while on zoom-out should open the patterns tab on the inserter', async ( {
+	test( 'Entering zoomed out mode zooms the canvas', async ( {
 		page,
+		editor,
 	} ) => {
-		// Trigger zoom out on Global Styles because there the inserter is not open.
-		await page.getByRole( 'button', { name: 'Styles' } ).click();
-		await page.getByRole( 'button', { name: 'Browse styles' } ).click();
-
-		// select the 1st pattern
-		await page
-			.frameLocator( 'iframe[name="editor-canvas"]' )
-			.locator( 'header' )
-			.click();
-
-		await expect( page.getByLabel( 'Add pattern' ) ).toHaveCount( 3 );
-		await page.getByLabel( 'Add pattern' ).first().click();
-		await expect( page.getByLabel( 'Add pattern' ) ).toHaveCount( 2 );
-
-		await expect(
-			page
-				.locator( '#tabs-2-allPatterns-view div' )
-				.filter( { hasText: 'All' } )
-				.nth( 1 )
-		).toBeVisible();
+		await page.getByLabel( 'Zoom Out' ).click();
+		const iframe = page.locator( 'iframe[name="editor-canvas"]' );
+		const html = editor.canvas.locator( 'html' );
+
+		//Check that the html is scaled
+		await expect( html ).toHaveCSS(
+			'transform',
+			'matrix(0.75, 0, 0, 0.75, 0, 0)'
+		);
+		const iframeRect = await iframe.boundingBox();
+		const htmlRect = await html.boundingBox();
+
+		//Check that the iframe is larger than the html
+		expect( iframeRect.width ).toBeGreaterThan( htmlRect.width );
+
+		//Check that the zoomed out content has a frame around it
+		expect( htmlRect.x ).toBeGreaterThan( iframeRect.x );
+
+		//We use border to separate the content from the frame so we need
+		//to check that that is present too, since boundingBox()
+		//includes the border to calculate the position
+		const borderWidths = await html.evaluate( ( el ) => {
+			const styles = window.getComputedStyle( el );
+			return {
+				top: parseFloat( styles.borderTopWidth ),
+				right: parseFloat( styles.borderRightWidth ),
+				bottom: parseFloat( styles.borderBottomWidth ),
+				left: parseFloat( styles.borderLeftWidth ),
+			};
+		} );
+		expect( htmlRect.y + borderWidths.top ).toBeGreaterThan( iframeRect.y );
```

Comment:
I saw that a change was just merged to switch this to padding - https://github.com/WordPress/gutenberg/pull/66012/files

If only `top` is used, maybe the other sides don't need to be returned from the `evaluate` callback.

Reply:
we are actually changing that right now, so yeah, I'll update once we have a final solution for that

Reply:
I updated this to use padding. It seems a bit brittle that the e2e test is dependent on implementation, but I can't think of a better way to test this